README.md

```diff
@@ -1,24 +1,200 @@
+  echo 'if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi' >> ~/.bash_profile
+  source ~/.bash_profile
+
+  # Install Ruby
+  rbenv install 3.0.1
+  rbenv global 3.0.1
+  ruby -v
+```
+
+This example shows how to install Ruby 3.0.1 which was the latest version in April 2021, but you can check to see if there is a newer version [here](https://www.ruby-lang.org/en/downloads/releases/). Iy is also assumed that you have [homebrew](https://brew.sh/) already installed on your Mac.
+
+Next you'll want to intall Rails and postgreSQL by running the following commands, but you can also use any SQL application of your choice
+
+```shell
+  gem install rails -v 7.0.6
+
+  sudo apt install postgresql libpq-dev
+```
+
+And start postgreSQL and the app with 
+
+```shell
+  sudo service postgresql start
+
+  rails server
+```
+
+### Setup
+
+Once you have ruby installed, run this command to get the project on your local machine.
+
+```sh
+  git clone git@github.com:jlvFlores/hello-rails-react.git
+```
+
+### Usage
+
+Once you have the project installed, access the project's root directory usign this command 
+
+```sh
+  cd hello-rails-react
+```
+
+And lastly, run this command to see the project in in your browser.
+
+```sh
+  ./bin/dev
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Here is a mention all of the collaborators of this project.
+
+👤 **Author**
+
+- GitHub: [@jlvFlores](https://github.com/jlvFlores)
+- Twitter: [@JoseVaz44312762](https://twitter.com/JoseVaz44312762)
+- LinkedIn: [Jose (Luis) Vazquez](https://www.linkedin.com/in/jose-luis-vazquez/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> The following are the future features that will be added to the project.
+
+- [ ] **Have more messages and a nice home page**
+- [ ] **It will probably not take over the world**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give me a hand by recommending me to potential employers! 😉🤝
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank those who have motivated me to keep on fighting despite how tough the journey may become.
+
+**I'd also like to mention [Gregoire Vella](https://www.behance.net/gregoirevella) and his [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from which I took inspiration for this project.**
```

Comment:
- [optional] Kindly remove the acknowledgement to behance if it is not relevant to your project. This will make your repo to look more professional. The same thing to the project title. 👍